app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,39 @@
+<nav class="navbar sticky-top navbar-expand-lg navbar-light bg-white">
+  <a class="navbar-brand ml-5" href="#"><i class='fas fa-dove'></i> SHIZU BnB</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
make this code by replacing this with the code provided below:
  `<%= link_to raw("<i class='fas fa-dove'></i> SHIZU BnB"), root_path, class: "navbar-brand ml-5" %>`